common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -953,6 +953,46 @@ def commits_details(base:, head:)
 
         it { is_expected.to eq("Bump the go_modules group across 2 directories with 2 updates") }
       end
+
+      context "with more then 5 dependencies" do
+        subject(:message) { builder.message }
+        let(:metadata) { { directory: "/foo" } }
+        let(:bar_dependencies) do
+          (1..5).map do |index|
+            Dependabot::Dependency.new(
+              name: "business#{index + 1}",
+              version: "#{index + 1}.5.0",
+              previous_version: "#{index + 1}.4.0",
+              package_manager: "dummy",
+              requirements: [],
+              previous_requirements: [],
+              metadata: { directory: "/bar" }
+            )
+          end
+        end
+        let(:dependencies) { bar_dependencies + [dependency] }
+
+        pr_message = "Bumps the go_modules group with 1 update in the /foo directory: " \
+                     "[business](https://github.com/gocardless/business).\n" \
+                     "Bumps the go_modules group with 5 updates in the /bar directory:\n\n" \
+                     "| Package | From | To |\n" \
+                     "| --- | --- | --- |\n" \
+                     "| [business2](https://github.com/gocardless/business2) | `2.4.0` | `2.5.0` |\n" \
+                     "| [business3](https://github.com/gocardless/business3) | `3.4.0` | `3.5.0` |\n" \
+                     "| [business4](https://github.com/gocardless/business4) | `4.4.0` | `4.5.0` |\n" \
+                     "| [business5](https://github.com/gocardless/business5) | `5.4.0` | `5.5.0` |\n" \
+                     "| [business6](https://github.com/gocardless/business6) | `6.4.0` | `6.5.0` |\n\n"
+
+        its(:pr_name) { should eq("Bump the go_modules group across 2 directories with 6 updates") }
+        it "contains the expected substring" do
+          expect(pr_message).to include("Bumps the go_modules group with 1 update in the /foo directory: " \
```

Comment:
I think you'll want to move this test somewhere under the `describe "#pr_message"` block below

Then you can do something like `expect(pr_message).to eq(expected_pr_message)`

Reply:
Done. Thanks for the review!